src/main.rs

```diff
@@ -59,12 +60,32 @@ use cli::generate_cli;
 use decompilation::*;
 use static_analysis::*;
 use results::*;
+use errors::*;
 pub use config::Config;
 pub use utils::*;
 
 static BANNER: &'static str = include_str!("banner.txt");
 
 fn main() {
+    if let Err(e) = run() {
+
+        for e in e.iter().skip(1) {
+            println!("caused by: {}", e);
+        }
+
+        // The backtrace is not always generated. Try to run this example
```

Comment:
Not sure of the usefulness of the backtrace here. In any case the dummy comment should be removed xD